docs/Tutorials/CORS.md

```diff
@@ -0,0 +1,93 @@
+
+# Cross-Origin Resource Sharing (CORS)
```

Comment:
I'd recommend just calling this `CORS`, otherwise the title in the sidebar spans two lines.

I'm also wondering if this could be under the Middleware folder, or a new Security folder 🤔

Reply:
Let me know I can change location. 
or better you can decide later befre the release